### DISCUSSION TITLE
Team Apps

### DISCUSSION THREAD

**Discussion by dbejic** [pcg-platinus/AM-Uebung#4] [Ideas]
Liste der Stakeholder

**Comment by ivonnemuellner** [comment]
Stakeholder |   | Kontaktdaten Ansprechpartner
-- | -- | --
Bewohner | Hausherr (AG u.PO) |  
  | Frau |  
  | Opa/Oma |  
  | Haustier |  
Smart Home Anbieter |   |  
Elektriker |   |  
Techniker |   |  
Gesetzgeber/Gemeinde |   |  
Nachbarn |   |  
Gerätehersteller |   |  
Freunde |   |  
Internetanbieter |   |  
Security/Polizei |   |  
Gäste/Fremdzugänge/Reinigungskraft |   |  
Versicherung |   |  
Installateur |   |  
Energieanbieter |   |  



</body>

</html>

**Comment by dbejic** [comment]
**Systemgrenze** 

**Innerhalb der Grenze**
- SMART Home Geräte (Dots, Kameras, Heizungssysteme etc.)
- Anwendungssoftware 
- Internetverbindung im Haus (WLAN)
- oder LAN-Verbindung im Haus
- Stromanbindung / Stromanschlüsse 


**Außerhalb der Grenze**
- Internetanbieter 
- Energieanbieter 
- Security

**Comment by christophgenis** [comment]
Zustandsdiagramm 

![Zustandsdiagramm_Apps](https://user-images.githubusercontent.com/125259486/218973544-4a433c17-7296-4510-9920-fca1269a5047.jpeg)
